packages/lib/RotatingLogs.test.ts

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
```

Comment:
Please have a look again at coding_style.md, section "Only import what you need". In general we should no longer import the whole fs package

Reply:
Also don't import from fs, use `fs-extra` instead